.github/workflows/Pipeline.yml

```diff
@@ -1,450 +1,116 @@
 name: Unit Testing, Coverage Collection, Package, Release, Documentation and Publish
 
-on: [ push ]
-
-defaults:
-  run:
-    shell: bash
+on: 
+  push:
+  workflow_dispatch:
 
 jobs:
 
-  UnitTesting:
-    name: ${{ matrix.icon }} Unit Tests using Python ${{ matrix.python }}
-    runs-on: ubuntu-latest
-
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-#          - {python: "3.6", icon: 🔴}  # until 23.12.2021
-          - {python: "3.7", icon: 🟠}  # until 27.06.2023
-          - {python: "3.8", icon: 🟡}  # until Oct. 2024
-          - {python: "3.9", icon: 🟢}  # until Oct. 2025
-          - {python: "3.10", icon: 🟢} # until Oct. 2026
-
-    env:
-      PYTHON: ${{ matrix.python }}
-    outputs:
-      python: ${{ env.PYTHON }}
-
-    steps:
-      - name: ⏬ Checkout repository
-        uses: actions/checkout@v2
+  Params:
+    uses: pyTooling/Actions/.github/workflows/Params.yml@v0
+    with:
+      name: pyEDAA.ProjectModel
+      python_version_list: '3.7 3.8 3.9 3.10'
```

Comment:
How is the default Python version set?
Is it fixed in the workflow job templates or is it the last element from the list?

Reply:
The default is set through `python_version`.
As discussed in a meeting, better be explicit, than use the last from the string.

Note: `>-` can be used for declaring each version in a row, but still have newlines and spaces trimmed by YAML.